src/modules/deviceSchemas.ts

```diff
@@ -31,3 +31,10 @@ export const getDeviceListSchema = z.object({
         .optional(),
 });
 export type GetDeviceList = z.infer<typeof getDeviceListSchema>;
+
+export const deleteDeviceSchema = z.object({
+    deviceId: z.string(),
+    deviceName: z.string(),
```

Comment:
why would you need deviceName to delete a device?

Reply:
deviceName is a type of device, like `TV`. deviceId is not an autogenerated field, and theoretically two different devices could have same id. Thats why index is `@@unique([deviceName, deviceId])`